### DISCUSSION TITLE
About consol in readme.MD

### DISCUSSION THREAD

**Discussion by uuc110** [charmbracelet/vhs#197] [Q&A] [answered]
Can anyone tell me what is the theme profile.json or anything like config for that consol in the readme section I really like it. So can I do it like that in my Windows Terminal? Please let me know if I can with its JSON file or configuration of the appearance

**Comment by maaslalani** [comment, marked as answer]
Hey @uuc110,

here is the JSON for the theme configuration:

```
{"name":"VHS","background":"#171717","foreground":"#dddddd","selection":"","cursor":"#dddddd","cursorAccent":"#171717","black":"#282a2e","brightBlack":"#4d4d4d","red":"#D74E6F","brightRed":"#FE5F86","green":"#31BB71","brightGreen":"#00D787","yellow":"#D3E561","brightYellow":"#EBFF71","blue":"#8056FF","brightBlue":"#9B79FF","magenta":"#ED61D7","brightMagenta":"#FF7AEA","cyan":"#04D7D7","brightCyan":"#00FEFE","white":"#bfbfbf","brightWhite":"#e6e6e6"}
```

Generated from https://go.dev/play/p/1rWHVXJjAVh
